FEM/pcs_dm.cpp

```diff
@@ -820,10 +820,10 @@ double CRFProcessDeformation::Execute(int loop_process_number)
                     "     "
                     "------------------------------------------------\n");
 
-                if (Error <= Tolerance_global_Newton &&
-                    Norm <= m_num->nls_abs_residual_tolerance &&
-                    NormU <= m_num->nls_abs_unknown_tolerance &&
-                    ErrorU <= m_num->nls_rel_unknown_tolerance)
+                if ((Error <= Tolerance_global_Newton &&
```

Comment:
For me, `Error <= Tolerance_global_Newton` is sufficient as a condition for convergence. Do we really need `&&` here?

Reply:
I dropped this change because it influences several benchmarks. I did this change because there is a case of an elastic problem occasionally solved by non-linear solver. In that case, the initial NormU is very small in a magnitude of 1.e-14 and non-linear solver cannot converged with that if-condition.